src/pages/social-feeds/index.vue

```diff
@@ -0,0 +1,545 @@
+        this.updateTopRankedCreators();
+      }
+    },
+    walletFollowees: {
+      immediate: true,
+      handler(walletFollowees) {
+        if (walletFollowees.length && !this.hasFetchedFollowees) {
+          this.fetchFolloweeWalletEvent();
+          this.hasFetchedFollowees = true;
+        }
+      },
+    },
+  },
+  mounted() {
+    this.currentMainTab = this.currentView;
+    if (this.getAddress) {
+      this.fetchNFTDisplayStateListByAddress(this.getAddress);
+      this.updateTopRankedCreators();
+    }
+  },
+  methods: {
+    ...mapActions(['lazyGetUserInfoByAddress']),
+    fetchUserInfo() {
+      this.lazyGetUserInfoByAddress(this.getAddress);
+    },
+    async updateTopRankedCreators() {
+      const res = await this.$axios.$get(
+        getCollectorTopRankedCreators(this.getAddress)
+      );
+      if (res.creators) {
+        let users = (await Promise.all(
+          // res.creators can be array of address, or array of { creator: address }
+          res.creators.map(c => this.lazyGetUserInfoByAddress(c.creator || c))
+        )).map((c, i) => ({
+          id: res.creators[i].creator || res.creators[i],
+          ...c,
+        }));
+        if (users && users.length > 10) {
+          users = fisherShuffle(users);
+          users = users.slice(0, 10);
+        }
+        this.topRankedUsers = users;
+      } else {
+        this.topRankedUsers = res.creators;
+      }
+    },
+    handleTopUserHover(i) {
+      logTrackerEvent(
+        this,
+        'MyDashboard',
+        `dashboard_top_faned_creators_hover`,
+        `${i}`,
+        1
+      );
+    },
+    handleTopUserClick(i) {
+      logTrackerEvent(
+        this,
+        'MyDashboard',
+        `dashboard_top_faned_creators_hover`,
+        `${i}`,
+        1
+      );
+    },
+    handleGoTown() {
+      this.$router.replace({
+        ...this.$route,
+        query: { view: TAB_OPTIONS.TOWN, tab: undefined },
+      });
+      this.handleCurrentTabChange(TAB_OPTIONS.TOWN);
+    },
+    handleGoCollectibles() {
+      this.$router.replace({
+        ...this.$route,
+        query: { view: TAB_OPTIONS.COLLECTIBLES, tab: 'collected' },
+      });
+      this.handleCurrentTabChange(TAB_OPTIONS.COLLECTIBLES);
+      this.loadNFTClassesForCurrentTabByAddress(this.getAddress);
+    },
+    handleGoUserStats() {
+      this.$router.replace({
+        ...this.$route,
+        query: { view: TAB_OPTIONS.USER_STATS, tab: undefined },
+      });
+      this.handleCurrentTabChange(TAB_OPTIONS.USER_STATS);
+    },
+    handleGoCollected() {
+      this.handleCollectiblesTabChange(tabOptions.collected);
+    },
+    handleGoCreated() {
+      this.handleCollectiblesTabChange(tabOptions.created);
+    },
+    handleCurrentTabChange(tab) {
+      switch (tab) {
+        case TAB_OPTIONS.TOWN:
+          logTrackerEvent(
+            this,
+            'SocialFeed',
+            'go_town_tab_clicked',
```

Comment:
For shorter name, we switched back to camelCase event name
```suggestion
            'FeedTabTownClick',
```